app/controllers/check_ins_controller.rb

```diff
@@ -33,6 +33,7 @@ def update
   end
 
   private def check_in_params
+    return {} unless params.key?(:check_in)
```

Comment:
We do `params.require(:check_in)`; which throws an exception if there is a query or posted form param for `check_in`.  We call `check_in_params` in the `new` action now, which almost never takes params (unless those params come from the plz check in naow email). This results in exceptions being thrown in the `new` action for no good reason. By using a guard clause to return an empty hash when params doesn't have a `check_in` key we prevent these exceptions from being thrown.